### ISSUE TITLE
All documents fail to show any text

### ISSUE THREAD

**Issue by nathcoad** [outline/outline#2427]
**To Reproduce**
1. Login to wiki, v 0.58.1
2. Home page listing recently viewed, no documents show
3. Click on any document and the title shows but no text

**Expected behavior**
Documents to load

**Screenshots**
![Screen Shot 2021-08-06 at 7 55 21 am](https://user-images.githubusercontent.com/16133259/128426456-d7f9eb83-8a31-46ea-af31-663aa22ad62a.png)

**Outline (please complete the following information):**
- Install: self hosted
- Version: 0.58.1

**Desktop (please complete the following information):**
 - OS: OS X 11.5.1
 - Browser Chrome
 - Version 92

Log file in docker shows this error:

> outline_postgres    | 2021-08-05 21:50:28.949 UTC [48] ERROR:  column document.previousTitles does not exist at character 3657
outline_postgres    | 2021-08-05 21:50:28.949 UTC [48] STATEMENT:  SELECT "document".*, "createdBy"."id" AS "createdBy.id", "createdBy"."email" AS "createdBy.email", "createdBy"."username" AS "createdBy.username", "createdBy"."name" AS "createdBy.name", "createdBy"."avatarUrl" AS "createdBy.avatarUrl", "createdBy"."isAdmin" AS "createdBy.isAdmin", "createdBy"."isViewer" AS "createdBy.isViewer", "createdBy"."service" AS "createdBy.service", "createdBy"."serviceId" AS "createdBy.serviceId", "createdBy"."jwtSecret" AS "createdBy.jwtSecret", "createdBy"."lastActiveAt" AS "createdBy.lastActiveAt", "createdBy"."lastActiveIp" AS "createdBy.lastActiveIp", "createdBy"."lastSignedInAt" AS "createdBy.lastSignedInAt", "createdBy"."lastSignedInIp" AS "createdBy.lastSignedInIp", "createdBy"."lastSigninEmailSentAt" AS "createdBy.lastSigninEmailSentAt", "createdBy"."suspendedAt" AS "createdBy.suspendedAt", "createdBy"."suspendedById" AS "createdBy.suspendedById", "createdBy"."language" AS "createdBy.language", "createdBy"."createdAt" AS "createdBy.createdAt", "createdBy"."updatedAt" AS "createdBy.updatedAt", "createdBy"."deletedAt" AS "createdBy.deletedAt", "createdBy"."teamId" AS "createdBy.teamId", "updatedBy"."id" AS "updatedBy.id", "updatedBy"."email" AS "updatedBy.email", "updatedBy"."username" AS "updatedBy.username", "updatedBy"."name" AS "updatedBy.name", "updatedBy"."avatarUrl" AS "updatedBy.avatarUrl", "updatedBy"."isAdmin" AS "updatedBy.isAdmin", "updatedBy"."isViewer" AS "updatedBy.isViewer", "updatedBy"."service" AS "updatedBy.service", "updatedBy"."serviceId" AS "updatedBy.serviceId", "updatedBy"."jwtSecret" AS "updatedBy.jwtSecret", "updatedBy"."lastActiveAt" AS "updatedBy.lastActiveAt", "updatedBy"."lastActiveIp" AS "updatedBy.lastActiveIp", "updatedBy"."lastSignedInAt" AS "updatedBy.lastSignedInAt", "updatedBy"."lastSignedInIp" AS "updatedBy.lastSignedInIp", "updatedBy"."lastSigninEmailSentAt" AS "updatedBy.lastSigninEmailSentAt", "updatedBy"."suspendedAt" AS "updatedBy.suspendedAt", "updatedBy"."suspendedById" AS "updatedBy.suspendedById", "updatedBy"."language" AS "updatedBy.language", "updatedBy"."createdAt" AS "updatedBy.createdAt", "updatedBy"."updatedAt" AS "updatedBy.updatedAt", "updatedBy"."deletedAt" AS "updatedBy.deletedAt", "updatedBy"."teamId" AS "updatedBy.teamId", "collection"."id" AS "collection.id", "collection"."urlId" AS "collection.urlId", "collection"."name" AS "collection.name", "collection"."description" AS "collection.description", "collection"."icon" AS "collection.icon", "collection"."color" AS "collection.color", "collection"."index" AS "collection.index", "collection"."permission" AS "collection.permission", "collection"."maintainerApprovalRequired" AS "collection.maintainerApprovalRequired", "collection"."documentStructure" AS "collection.documentStructure", "collection"."sharing" AS "collection.sharing", "collection"."sort" AS "collection.sort", "collection"."createdAt" AS "collection.createdAt", "collection"."updatedAt" AS "collection.updatedAt", "collection"."deletedAt" AS "collection.deletedAt", "collection"."createdById" AS "collection.createdById", "collection"."teamId" AS "collection.teamId", "collection->memberships"."permission" AS "collection.memberships.permission", "collection->memberships"."createdAt" AS "collection.memberships.createdAt", "collection->memberships"."updatedAt" AS "collection.memberships.updatedAt", "collection->memberships"."collectionId" AS "collection.memberships.collectionId", "collection->memberships"."userId" AS "collection.memberships.userId", "collection->memberships"."createdById" AS "collection.memberships.createdById" FROM (SELECT "document"."id", "document"."urlId", "document"."title", "document"."previousTitles", "document"."version", "document"."template", "document"."editorVersion", "document"."text", "document"."isWelcome", "document"."revisionCount", "document"."archivedAt", "document"."publishedAt", "document"."parentDocumentId", "document"."collaboratorIds", "document"."createdAt", "document"."updatedAt", "document"."deletedAt", "document"."collectionId", "document"."teamId", "document"."templateId", "document"."createdById", "document"."lastModifiedById", "document"."pinnedById", "document"."userId" FROM "documents" AS "document" WHERE ("document"."deletedAt" IS NULL AND ("document"."publishedAt" IS NULL AND "document"."userId" = 'e55504cd-1815-42d4-b1f8-a6f621ca1056' AND "document"."collectionId" IN ('eb60d902-78a8-4672-a637-f43fe628ba62', '12c36b03-1a27-4cad-a66d-6d4fbd3a5c38', '14556459-ca43-4a46-a9b8-7fc3dd675d72', '1414b3df-ce7c-414c-9bfc-6063d7bb7001'))) ORDER BY "document"."updatedAt" DESC LIMIT 15 OFFSET 0) AS "document" LEFT OUTER JOIN "users" AS "createdBy" ON "document"."createdById" = "createdBy"."id" LEFT OUTER JOIN "users" AS "updatedBy" ON "document"."lastModifiedById" = "updatedBy"."id" LEFT OUTER JOIN "collections" AS "collection" ON "document"."collectionId" = "collection"."id" AND ("collection"."deletedAt" IS NULL) LEFT OUTER JOIN "collection_users" AS "collection->memberships" ON "collection"."id" = "collection->memberships"."collectionId" AND "collection->memberships"."userId" = 'e55504cd-1815-42d4-b1f8-a6f621ca1056' ORDER BY "document"."updatedAt" DESC;
outline             | DatabaseError [SequelizeDatabaseError]: column document.previousTitles does not exist
outline             |     at Query.formatError (/opt/outline/node_modules/sequelize/lib/dialects/postgres/query.js:386:16)
outline             |     at Query.run (/opt/outline/node_modules/sequelize/lib/dialects/postgres/query.js:87:18)
outline             |     at processTicksAndRejections (internal/process/task_queues.js:95:5)
outline             |     at async /opt/outline/node_modules/sequelize/lib/sequelize.js:619:16
outline             |     at async PostgresQueryInterface.select (/opt/outline/node_modules/sequelize/lib/dialects/abstract/query-interface.js:938:12)
outline             |     at async Function.findAll (/opt/outline/node_modules/sequelize/lib/model.js:1753:21)
outline             |     at async /opt/outline/build/server/api/documents.js:434:21
outline             |     at async apiWrapperMiddleware (/opt/outline/build/server/api/middlewares/apiWrapper.js:14:5)
outline             |     at async errorHandlingMiddleware (/opt/outline/build/server/middlewares/errorHandling.js:17:7)
outline             |     at async /opt/outline/node_modules/koa-mount/index.js:58:5
outline             |     at async /opt/outline/node_modules/koa-mount/index.js:52:26
outline             |     at async logger (/opt/outline/node_modules/koa-logger/index.js:67:7) {
outline             |   parent: error: column document.previousTitles does not exist
outline             |       at Parser.parseErrorMessage (/opt/outline/node_modules/pg-protocol/dist/parser.js:278:15)
outline             |       at Parser.handlePacket (/opt/outline/node_modules/pg-protocol/dist/parser.js:126:29)
outline             |       at Parser.parse (/opt/outline/node_modules/pg-protocol/dist/parser.js:39:38)
outline             |       at Socket.<anonymous> (/opt/outline/node_modules/pg-protocol/dist/index.js:10:42)
outline             |       at Socket.emit (events.js:400:28)
outline             |       at Socket.emit (domain.js:470:12)
outline             |       at addChunk (internal/streams/readable.js:290:12)
outline             |       at readableAddChunk (internal/streams/readable.js:265:9)
outline             |       at Socket.Readable.push (internal/streams/readable.js:204:10)
outline             |       at TCP.onStreamRead (internal/stream_base_commons.js:188:23) {
outline             |     length: 123,
outline             |     severity: 'ERROR',
outline             |     code: '42703',
outline             |     detail: undefined,
outline             |     hint: undefined,
outline             |     position: '3657',
outline             |     internalPosition: undefined,
outline             |     internalQuery: undefined,
outline             |     where: undefined,
outline             |     schema: undefined,
outline             |     table: undefined,
outline             |     column: undefined,
outline             |     dataType: undefined,
outline             |     constraint: undefined,
outline             |     file: 'parse_relation.c',
outline             |     line: '3514',
outline             |     routine: 'errorMissingColumn',
outline             |     sql: `SELECT "document".*, "createdBy"."id" AS "createdBy.id", "createdBy"."email" AS "createdBy.email", "createdBy"."username" AS "createdBy.username", "createdBy"."name" AS "createdBy.name", "createdBy"."avatarUrl" AS "createdBy.avatarUrl", "createdBy"."isAdmin" AS "createdBy.isAdmin", "createdBy"."isViewer" AS "createdBy.isViewer", "createdBy"."service" AS "createdBy.service", "createdBy"."serviceId" AS "createdBy.serviceId", "createdBy"."jwtSecret" AS "createdBy.jwtSecret", "createdBy"."lastActiveAt" AS "createdBy.lastActiveAt", "createdBy"."lastActiveIp" AS "createdBy.lastActiveIp", "createdBy"."lastSignedInAt" AS "createdBy.lastSignedInAt", "createdBy"."lastSignedInIp" AS "createdBy.lastSignedInIp", "createdBy"."lastSigninEmailSentAt" AS "createdBy.lastSigninEmailSentAt", "createdBy"."suspendedAt" AS "createdBy.suspendedAt", "createdBy"."suspendedById" AS "createdBy.suspendedById", "createdBy"."language" AS "createdBy.language", "createdBy"."createdAt" AS "createdBy.createdAt", "createdBy"."updatedAt" AS "createdBy.updatedAt", "createdBy"."deletedAt" AS "createdBy.deletedAt", "createdBy"."teamId" AS "createdBy.teamId", "updatedBy"."id" AS "updatedBy.id", "updatedBy"."email" AS "updatedBy.email", "updatedBy"."username" AS "updatedBy.username", "updatedBy"."name" AS "updatedBy.name", "updatedBy"."avatarUrl" AS "updatedBy.avatarUrl", "updatedBy"."isAdmin" AS "updatedBy.isAdmin", "updatedBy"."isViewer" AS "updatedBy.isViewer", "updatedBy"."service" AS "updatedBy.service", "updatedBy"."serviceId" AS "updatedBy.serviceId", "updatedBy"."jwtSecret" AS "updatedBy.jwtSecret", "updatedBy"."lastActiveAt" AS "updatedBy.lastActiveAt", "updatedBy"."lastActiveIp" AS "updatedBy.lastActiveIp", "updatedBy"."lastSignedInAt" AS "updatedBy.lastSignedInAt", "updatedBy"."lastSignedInIp" AS "updatedBy.lastSignedInIp", "updatedBy"."lastSigninEmailSentAt" AS "updatedBy.lastSigninEmailSentAt", "updatedBy"."suspendedAt" AS "updatedBy.suspendedAt", "updatedBy"."suspendedById" AS "updatedBy.suspendedById", "updatedBy"."language" AS "updatedBy.language", "updatedBy"."createdAt" AS "updatedBy.createdAt", "updatedBy"."updatedAt" AS "updatedBy.updatedAt", "updatedBy"."deletedAt" AS "updatedBy.deletedAt", "updatedBy"."teamId" AS "updatedBy.teamId", "collection"."id" AS "collection.id", "collection"."urlId" AS "collection.urlId", "collection"."name" AS "collection.name", "collection"."description" AS "collection.description", "collection"."icon" AS "collection.icon", "collection"."color" AS "collection.color", "collection"."index" AS "collection.index", "collection"."permission" AS "collection.permission", "collection"."maintainerApprovalRequired" AS "collection.maintainerApprovalRequired", "collection"."documentStructure" AS "collection.documentStructure", "collection"."sharing" AS "collection.sharing", "collection"."sort" AS "collection.sort", "collection"."createdAt" AS "collection.createdAt", "collection"."updatedAt" AS "collection.updatedAt", "collection"."deletedAt" AS "collection.deletedAt", "collection"."createdById" AS "collection.createdById", "collection"."teamId" AS "collection.teamId", "collection->memberships"."permission" AS "collection.memberships.permission", "collection->memberships"."createdAt" AS "collection.memberships.createdAt", "collection->memberships"."updatedAt" AS "collection.memberships.updatedAt", "collection->memberships"."collectionId" AS "collection.memberships.collectionId", "collection->memberships"."userId" AS "collection.memberships.userId", "collection->memberships"."createdById" AS "collection.memberships.createdById" FROM (SELECT "document"."id", "document"."urlId", "document"."title", "document"."previousTitles", "document"."version", "document"."template", "document"."editorVersion", "document"."text", "document"."isWelcome", "document"."revisionCount", "document"."archivedAt", "document"."publishedAt", "document"."parentDocumentId", "document"."collaboratorIds", "document"."createdAt", "document"."updatedAt", "document"."deletedAt", "document"."collectionId", "document"."teamId", "document"."templateId", "document"."createdById", "document"."lastModifiedById", "document"."pinnedById", "document"."userId" FROM "documents" AS "document" WHERE ("document"."deletedAt" IS NULL AND ("document"."publishedAt" IS NULL AND "document"."userId" = 'e55504cd-1815-42d4-b1f8-a6f621ca1056' AND "document"."collectionId" IN ('eb60d902-78a8-4672-a637-f43fe628ba62', '12c36b03-1a27-4cad-a66d-6d4fbd3a5c38', '14556459-ca43-4a46-a9b8-7fc3dd675d72', '1414b3df-ce7c-414c-9bfc-6063d7bb7001'))) ORDER BY "document"."updatedAt" DESC LIMIT 15 OFFSET 0) AS "document" LEFT OUTER JOIN "users" AS "createdBy" ON "document"."createdById" = "createdBy"."id" LEFT OUTER JOIN "users" AS "updatedBy" ON "document"."lastModifiedById" = "updatedBy"."id" LEFT OUTER JOIN "collections" AS "collection" ON "document"."collectionId" = "collection"."id" AND ("collection"."deletedAt" IS NULL) LEFT OUTER JOIN "collection_users" AS "collection->memberships" ON "collection"."id" = "collection->memberships"."collectionId" AND "collection->memberships"."userId" = 'e55504cd-1815-42d4-b1f8-a6f621ca1056' ORDER BY "document"."updatedAt" DESC;`,
outline             |     parameters: undefined
outline             |   },
outline             |   original: error: column document.previousTitles does not exist
outline             |       at Parser.parseErrorMessage (/opt/outline/node_modules/pg-protocol/dist/parser.js:278:15)
outline             |       at Parser.handlePacket (/opt/outline/node_modules/pg-protocol/dist/parser.js:126:29)
outline             |       at Parser.parse (/opt/outline/node_modules/pg-protocol/dist/parser.js:39:38)
outline             |       at Socket.<anonymous> (/opt/outline/node_modules/pg-protocol/dist/index.js:10:42)
outline             |       at Socket.emit (events.js:400:28)
outline             |       at Socket.emit (domain.js:470:12)
outline             |       at addChunk (internal/streams/readable.js:290:12)
outline             |       at readableAddChunk (internal/streams/readable.js:265:9)
outline             |       at Socket.Readable.push (internal/streams/readable.js:204:10)
outline             |       at TCP.onStreamRead (internal/stream_base_commons.js:188:23) {
outline             |     length: 123,
outline             |     severity: 'ERROR',
outline             |     code: '42703',
outline             |     detail: undefined,
outline             |     hint: undefined,
outline             |     position: '3657',
outline             |     internalPosition: undefined,
outline             |     internalQuery: undefined,
outline             |     where: undefined,
outline             |     schema: undefined,
outline             |     table: undefined,
outline             |     column: undefined,
outline             |     dataType: undefined,
outline             |     constraint: undefined,
outline             |     file: 'parse_relation.c',
outline             |     line: '3514',
outline             |     routine: 'errorMissingColumn',
outline             |     sql: `SELECT "document".*, "createdBy"."id" AS "createdBy.id", "createdBy"."email" AS "createdBy.email", "createdBy"."username" AS "createdBy.username", "createdBy"."name" AS "createdBy.name", "createdBy"."avatarUrl" AS "createdBy.avatarUrl", "createdBy"."isAdmin" AS "createdBy.isAdmin", "createdBy"."isViewer" AS "createdBy.isViewer", "createdBy"."service" AS "createdBy.service", "createdBy"."serviceId" AS "createdBy.serviceId", "createdBy"."jwtSecret" AS "createdBy.jwtSecret", "createdBy"."lastActiveAt" AS "createdBy.lastActiveAt", "createdBy"."lastActiveIp" AS "createdBy.lastActiveIp", "createdBy"."lastSignedInAt" AS "createdBy.lastSignedInAt", "createdBy"."lastSignedInIp" AS "createdBy.lastSignedInIp", "createdBy"."lastSigninEmailSentAt" AS "createdBy.lastSigninEmailSentAt", "createdBy"."suspendedAt" AS "createdBy.suspendedAt", "createdBy"."suspendedById" AS "createdBy.suspendedById", "createdBy"."language" AS "createdBy.language", "createdBy"."createdAt" AS "createdBy.createdAt", "createdBy"."updatedAt" AS "createdBy.updatedAt", "createdBy"."deletedAt" AS "createdBy.deletedAt", "createdBy"."teamId" AS "createdBy.teamId", "updatedBy"."id" AS "updatedBy.id", "updatedBy"."email" AS "updatedBy.email", "updatedBy"."username" AS "updatedBy.username", "updatedBy"."name" AS "updatedBy.name", "updatedBy"."avatarUrl" AS "updatedBy.avatarUrl", "updatedBy"."isAdmin" AS "updatedBy.isAdmin", "updatedBy"."isViewer" AS "updatedBy.isViewer", "updatedBy"."service" AS "updatedBy.service", "updatedBy"."serviceId" AS "updatedBy.serviceId", "updatedBy"."jwtSecret" AS "updatedBy.jwtSecret", "updatedBy"."lastActiveAt" AS "updatedBy.lastActiveAt", "updatedBy"."lastActiveIp" AS "updatedBy.lastActiveIp", "updatedBy"."lastSignedInAt" AS "updatedBy.lastSignedInAt", "updatedBy"."lastSignedInIp" AS "updatedBy.lastSignedInIp", "updatedBy"."lastSigninEmailSentAt" AS "updatedBy.lastSigninEmailSentAt", "updatedBy"."suspendedAt" AS "updatedBy.suspendedAt", "updatedBy"."suspendedById" AS "updatedBy.suspendedById", "updatedBy"."language" AS "updatedBy.language", "updatedBy"."createdAt" AS "updatedBy.createdAt", "updatedBy"."updatedAt" AS "updatedBy.updatedAt", "updatedBy"."deletedAt" AS "updatedBy.deletedAt", "updatedBy"."teamId" AS "updatedBy.teamId", "collection"."id" AS "collection.id", "collection"."urlId" AS "collection.urlId", "collection"."name" AS "collection.name", "collection"."description" AS "collection.description", "collection"."icon" AS "collection.icon", "collection"."color" AS "collection.color", "collection"."index" AS "collection.index", "collection"."permission" AS "collection.permission", "collection"."maintainerApprovalRequired" AS "collection.maintainerApprovalRequired", "collection"."documentStructure" AS "collection.documentStructure", "collection"."sharing" AS "collection.sharing", "collection"."sort" AS "collection.sort", "collection"."createdAt" AS "collection.createdAt", "collection"."updatedAt" AS "collection.updatedAt", "collection"."deletedAt" AS "collection.deletedAt", "collection"."createdById" AS "collection.createdById", "collection"."teamId" AS "collection.teamId", "collection->memberships"."permission" AS "collection.memberships.permission", "collection->memberships"."createdAt" AS "collection.memberships.createdAt", "collection->memberships"."updatedAt" AS "collection.memberships.updatedAt", "collection->memberships"."collectionId" AS "collection.memberships.collectionId", "collection->memberships"."userId" AS "collection.memberships.userId", "collection->memberships"."createdById" AS "collection.memberships.createdById" FROM (SELECT "document"."id", "document"."urlId", "document"."title", "document"."previousTitles", "document"."version", "document"."template", "document"."editorVersion", "document"."text", "document"."isWelcome", "document"."revisionCount", "document"."archivedAt", "document"."publishedAt", "document"."parentDocumentId", "document"."collaboratorIds", "document"."createdAt", "document"."updatedAt", "document"."deletedAt", "document"."collectionId", "document"."teamId", "document"."templateId", "document"."createdById", "document"."lastModifiedById", "document"."pinnedById", "document"."userId" FROM "documents" AS "document" WHERE ("document"."deletedAt" IS NULL AND ("document"."publishedAt" IS NULL AND "document"."userId" = 'e55504cd-1815-42d4-b1f8-a6f621ca1056' AND "document"."collectionId" IN ('eb60d902-78a8-4672-a637-f43fe628ba62', '12c36b03-1a27-4cad-a66d-6d4fbd3a5c38', '14556459-ca43-4a46-a9b8-7fc3dd675d72', '1414b3df-ce7c-414c-9bfc-6063d7bb7001'))) ORDER BY "document"."updatedAt" DESC LIMIT 15 OFFSET 0) AS "document" LEFT OUTER JOIN "users" AS "createdBy" ON "document"."createdById" = "createdBy"."id" LEFT OUTER JOIN "users" AS "updatedBy" ON "document"."lastModifiedById" = "updatedBy"."id" LEFT OUTER JOIN "collections" AS "collection" ON "document"."collectionId" = "collection"."id" AND ("collection"."deletedAt" IS NULL) LEFT OUTER JOIN "collection_users" AS "collection->memberships" ON "collection"."id" = "collection->memberships"."collectionId" AND "collection->memberships"."userId" = 'e55504cd-1815-42d4-b1f8-a6f621ca1056' ORDER BY "document"."updatedAt" DESC;`,
outline             |     parameters: undefined
outline             |   },
outline             |   sql: `SELECT "document".*, "createdBy"."id" AS "createdBy.id", "createdBy"."email" AS "createdBy.email", "createdBy"."username" AS "createdBy.username", "createdBy"."name" AS "createdBy.name", "createdBy"."avatarUrl" AS "createdBy.avatarUrl", "createdBy"."isAdmin" AS "createdBy.isAdmin", "createdBy"."isViewer" AS "createdBy.isViewer", "createdBy"."service" AS "createdBy.service", "createdBy"."serviceId" AS "createdBy.serviceId", "createdBy"."jwtSecret" AS "createdBy.jwtSecret", "createdBy"."lastActiveAt" AS "createdBy.lastActiveAt", "createdBy"."lastActiveIp" AS "createdBy.lastActiveIp", "createdBy"."lastSignedInAt" AS "createdBy.lastSignedInAt", "createdBy"."lastSignedInIp" AS "createdBy.lastSignedInIp", "createdBy"."lastSigninEmailSentAt" AS "createdBy.lastSigninEmailSentAt", "createdBy"."suspendedAt" AS "createdBy.suspendedAt", "createdBy"."suspendedById" AS "createdBy.suspendedById", "createdBy"."language" AS "createdBy.language", "createdBy"."createdAt" AS "createdBy.createdAt", "createdBy"."updatedAt" AS "createdBy.updatedAt", "createdBy"."deletedAt" AS "createdBy.deletedAt", "createdBy"."teamId" AS "createdBy.teamId", "updatedBy"."id" AS "updatedBy.id", "updatedBy"."email" AS "updatedBy.email", "updatedBy"."username" AS "updatedBy.username", "updatedBy"."name" AS "updatedBy.name", "updatedBy"."avatarUrl" AS "updatedBy.avatarUrl", "updatedBy"."isAdmin" AS "updatedBy.isAdmin", "updatedBy"."isViewer" AS "updatedBy.isViewer", "updatedBy"."service" AS "updatedBy.service", "updatedBy"."serviceId" AS "updatedBy.serviceId", "updatedBy"."jwtSecret" AS "updatedBy.jwtSecret", "updatedBy"."lastActiveAt" AS "updatedBy.lastActiveAt", "updatedBy"."lastActiveIp" AS "updatedBy.lastActiveIp", "updatedBy"."lastSignedInAt" AS "updatedBy.lastSignedInAt", "updatedBy"."lastSignedInIp" AS "updatedBy.lastSignedInIp", "updatedBy"."lastSigninEmailSentAt" AS "updatedBy.lastSigninEmailSentAt", "updatedBy"."suspendedAt" AS "updatedBy.suspendedAt", "updatedBy"."suspendedById" AS "updatedBy.suspendedById", "updatedBy"."language" AS "updatedBy.language", "updatedBy"."createdAt" AS "updatedBy.createdAt", "updatedBy"."updatedAt" AS "updatedBy.updatedAt", "updatedBy"."deletedAt" AS "updatedBy.deletedAt", "updatedBy"."teamId" AS "updatedBy.teamId", "collection"."id" AS "collection.id", "collection"."urlId" AS "collection.urlId", "collection"."name" AS "collection.name", "collection"."description" AS "collection.description", "collection"."icon" AS "collection.icon", "collection"."color" AS "collection.color", "collection"."index" AS "collection.index", "collection"."permission" AS "collection.permission", "collection"."maintainerApprovalRequired" AS "collection.maintainerApprovalRequired", "collection"."documentStructure" AS "collection.documentStructure", "collection"."sharing" AS "collection.sharing", "collection"."sort" AS "collection.sort", "collection"."createdAt" AS "collection.createdAt", "collection"."updatedAt" AS "collection.updatedAt", "collection"."deletedAt" AS "collection.deletedAt", "collection"."createdById" AS "collection.createdById", "collection"."teamId" AS "collection.teamId", "collection->memberships"."permission" AS "collection.memberships.permission", "collection->memberships"."createdAt" AS "collection.memberships.createdAt", "collection->memberships"."updatedAt" AS "collection.memberships.updatedAt", "collection->memberships"."collectionId" AS "collection.memberships.collectionId", "collection->memberships"."userId" AS "collection.memberships.userId", "collection->memberships"."createdById" AS "collection.memberships.createdById" FROM (SELECT "document"."id", "document"."urlId", "document"."title", "document"."previousTitles", "document"."version", "document"."template", "document"."editorVersion", "document"."text", "document"."isWelcome", "document"."revisionCount", "document"."archivedAt", "document"."publishedAt", "document"."parentDocumentId", "document"."collaboratorIds", "document"."createdAt", "document"."updatedAt", "document"."deletedAt", "document"."collectionId", "document"."teamId", "document"."templateId", "document"."createdById", "document"."lastModifiedById", "document"."pinnedById", "document"."userId" FROM "documents" AS "document" WHERE ("document"."deletedAt" IS NULL AND ("document"."publishedAt" IS NULL AND "document"."userId" = 'e55504cd-1815-42d4-b1f8-a6f621ca1056' AND "document"."collectionId" IN ('eb60d902-78a8-4672-a637-f43fe628ba62', '12c36b03-1a27-4cad-a66d-6d4fbd3a5c38', '14556459-ca43-4a46-a9b8-7fc3dd675d72', '1414b3df-ce7c-414c-9bfc-6063d7bb7001'))) ORDER BY "document"."updatedAt" DESC LIMIT 15 OFFSET 0) AS "document" LEFT OUTER JOIN "users" AS "createdBy" ON "document"."createdById" = "createdBy"."id" LEFT OUTER JOIN "users" AS "updatedBy" ON "document"."lastModifiedById" = "updatedBy"."id" LEFT OUTER JOIN "collections" AS "collection" ON "document"."collectionId" = "collection"."id" AND ("collection"."deletedAt" IS NULL) LEFT OUTER JOIN "collection_users" AS "collection->memberships" ON "collection"."id" = "collection->memberships"."collectionId" AND "collection->memberships"."userId" = 'e55504cd-1815-42d4-b1f8-a6f621ca1056' ORDER BY "document"."updatedAt" DESC;`,
outline             |   parameters: undefined
outline             | }

**Comment by tommoor** [comment]
You must always run migrations when updating, eg:

`docker run --rm outlinewiki/outline yarn db:migrate`

or 

`docker run --rm outlinewiki/outline yarn db:migrate --env=production-ssl-disabled` if not using SSL

**Comment by nathcoad** [comment]
Last time I upgraded, I did run those migrations. Outline was working fine a couple of days ago, and nothing has changed since then. No restarts of the container, no updates etc. I've also tried logging out and back in again, and a different browser (safari). 
I'll try running the migration again and see if something was missed the first time around.

**Comment by tommoor** [comment]
The error clearly shows a missing database column that was added in the `0.58.X` release, so it's definitely the problem – as to _why_ it became an issue for you now I can't say :)

**Comment by nathcoad** [comment]
Last time I modified the command in my docker-compose file to `command: yarn db:migrate --env production-ssl-disabled`, started the container, let it run the migration, stopped it and changed the command line back to `command: yarn start --env=production-ssl-disabled`

I did the same thing just now, got this output:

`
outline             | yarn run v1.22.5
outline             | $ sequelize db:migrate --env production-ssl-disabled
outline             |
outline             | Sequelize CLI [Node: 14.17.4, CLI: 6.2.0, ORM: 6.3.5]
outline             |
outline             | Loaded configuration file "server/config/database.json".
outline             | Using environment "production-ssl-disabled".
outline             | == 20210716064654-introduce-previousTitles: migrating =======
outline             | == 20210716064654-introduce-previousTitles: migrated (0.087s)
outline             |
outline             | == 20210716071454-search-index-previousTitles: migrating =======
outline             | == 20210716071454-search-index-previousTitles: migrated (0.056s)
outline             |
outline             | == 20210716162923-events-indexes: migrating =======
outline             | == 20210716162923-events-indexes: migrated (0.161s)
outline             |
outline             | Done in 3.21s.
outline exited with code 0
`
And started the container back up with the normal command, and everything looks to be working again just fine. Thanks so much for your super fast response!

As to why the migration didn't work fully last time, that is a bit odd but I guess your advice did the trick.
